### PR TITLE
Recurring contributions final project polishing

### PR DESCRIPTION
**Pull request by sbinlondon**

### CODE REVIEW

File: components/CollectiveNavbar.js [hunk 1]
```diff
@@ -192,6 +192,7 @@ const DEFAULT_SECTIONS = {
     Sections.CONVERSATIONS,
     Sections.BUDGET,
     Sections.CONTRIBUTORS,
+    Sections.RECURRING_CONTRIBUTIONS,
```
Comment (Betree) [inline components/CollectiveNavbar.js:195]: Do we really want to show this section for all collectives?

File: components/recurring-contributions/RecurringContributionsCard.js [hunk 2]
```diff
@@ -73,15 +69,13 @@ const RecurringContributionsCard = ({ collective, status, contribution, createNo
...
-  // const userIsAdminOfCollectiveOrOrg
-  const userIsAdmin = userIsLoggedInUser; // || userIsAdminOfCollectiveOrOrg
+  const isAdmin = LoggedInUser && LoggedInUser.canEditCollective(account);
 
   return (
     <StyledCard onMouseEnter={() => setHovering(true)} onMouseLeave={() => setHovering(false)} {...props}>
       <Container style={{ background: getBackground(collective) }} backgroundSize="cover" height={100} px={3} pt={26}>
         <Container border="2px solid white" borderRadius="25%" backgroundColor="white.full" width={68}>
-          <LinkCollective collective={collective}>
+          <LinkCollective collective={collective} type={collective.type}>
```
Comment (Betree) [inline components/recurring-contributions/RecurringContributionsCard.js:78]: I don't see any `type` prop listed for `LinkCollective`, what's its purpose?

Comment (sbinlondon) [reply components/recurring-contributions/RecurringContributionsCard.js:78]: There was this error. I think I have misunderstood it then?
image

Comment (Betree) [reply components/recurring-contributions/RecurringContributionsCard.js:78]: The missing prop is `collective.type`, not `type`. May be the field is not fetched in the GraphQL query? In any case I'd be surprised if this change removed the warning

Comment (sbinlondon) [reply components/recurring-contributions/RecurringContributionsCard.js:78]: You were right! It was adding the graphql field, not the prop, that fixed this.

File: components/CollectiveNavbar.js (hunk 1 above)
Comment (sbinlondon) [reply components/CollectiveNavbar.js:195]: I'm not sure. If you let me know which cases we wouldn't want to show it, I can add some logic?

Comment (Betree) [reply components/CollectiveNavbar.js:195]: Most collectives don't have any recurring contributions, we probably don't want to add it on https://opencollective.com/babel/ for example. An idea would be to show that for collectives that have at least one active recurring contribution. I just checked, and there's only 43 in the database.


```sql
SELECT * FROM "Subscriptions" s
INNER JOIN "Orders" o ON o."SubscriptionId" = s.id
INNER JOIN "Collectives" c ON o."FromCollectiveId" = c.id
WHERE s."isActive" IS TRUE
AND c."type" = 'COLLECTIVE'
```

**Pull request closed without merging**